assets/app/views/directives/_truncate.html

```diff
@@ -0,0 +1,4 @@
+<span
```

Comment:
since we're not inlining templates (yet) in our build, maybe inline this?
if you take `content` as input, and always set `visibleContent` and `truncated` in the directive's watch, this can simplify to 
```
<span ng-attr-title="{{content}}">{{visibleContent}}<span ng-if="{{truncated}}">&hellip;</span></span>
```